tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -5017,4 +5019,147 @@ public function test_add_email_currency_conversion_notice_outputs_plain_text_for
+		$checkout_session_id = 'cs_test_eea_notice_1';
+		$order_helper        = WC_Stripe_Order_Helper::get_instance();
+		$order_helper->update_stripe_checkout_session_id( $order, $checkout_session_id );
+		$order_helper->update_stripe_presentment_amount( $order, 1500 );
+		$order_helper->update_stripe_presentment_currency( $order, 'eur' );
+
+		ob_start();
+		$this->mock_gateway->add_currency_conversion_notice( $order );
+		$output = ob_get_clean();
+
+		$this->assertStringContainsString( 'European Central Bank (ECB) interbank rate', $output );
+	}
```

PHPMD (2.15.0)

[error] The method test_add_currency_conversion_notice_appends_ecb_fee_for_eea_customer is not named in camelCase. (undefined)

(CamelCaseMethodName)